packages/client/src/components/reports/scape/ScapeExecutionsResult.tsx

```diff
@@ -121,71 +125,91 @@ export default class ScapeExecutionsResult extends React.Component<Props, State>
+                  groupScoreByMetricGroupNameMap={groupScoreByMetricGroupNameMap as KvMap<CanaryJudgeGroupScore>}
+                  metricSetPairsByIdMap={metricSetPairsByIdMap as KvMap<MetricSetPair>}
+                  classificationCountMap={classificationCountMap as Map<string, number>}
+                  metricGroupNamesDescByWeight={metricGroupNamesDescByWeight as string[]}
+                  displayMetricOverview={displayMetricOverview as boolean}
+                  handleOverviewSelection={handleOverviewSelection}
+                  handleMetricSelection={handleMetricSelection}
+                />
+              )}
+            </div>
           </div>
-          <div className="scape-executions-result-wrapper">
-            {isTerminalFailure(selectedCanaryExecutionResult) ? (
-              <div className="terminal-canary-wrapper">
-                <TerminalResult exception={selectedCanaryExecutionResult.exception} />
-              </div>
-            ) : (
-              <ScapeRunResult
-                application={application as string}
-                user={user as string}
-                metricSourceType={metricSourceType as string}
-                metricsAccountName={metricsAccountName as string}
-                storageAccountName={storageAccountName as string}
-                thresholds={thresholds as CanaryClassifierThresholdsConfig}
-                lifetime={lifetime as number}
-                selectedCanaryExecutionResult={selectedCanaryExecutionResult as CanaryExecutionResult}
-                result={result as CanaryResult}
-                totalRuns={results.canaryScores.length as number}
-                selectedRunNumber={selectedRunNumber as number}
-                selectedMetric={selectedMetric as string}
-                canaryConfig={canaryConfig as CanaryConfig}
-                canaryAnalysisResultByIdMap={canaryAnalysisResultByIdMap as KvMap<CanaryAnalysisResult>}
-                idListByMetricGroupNameMap={idListByMetricGroupNameMap as KvMap<string[]>}
-                groupScoreByMetricGroupNameMap={groupScoreByMetricGroupNameMap as KvMap<CanaryJudgeGroupScore>}
-                metricSetPairsByIdMap={metricSetPairsByIdMap as KvMap<MetricSetPair>}
-                classificationCountMap={classificationCountMap as Map<string, number>}
-                metricGroupNamesDescByWeight={metricGroupNamesDescByWeight as string[]}
-                displayMetricOverview={displayMetricOverview as boolean}
-                handleOverviewSelection={handleOverviewSelection}
-                handleMetricSelection={handleMetricSelection}
-              />
-            )}
+
+        :
+          <div className="terminal-canary-wrapper">
+            <TerminalResult
+                exception={ofNullable(exception).orElse(
+                    {
+                            "details":
+                              {
+                                "error": "The canary terminated without an error message",
```

Comment:
To be clear, it would be helpful to add the execution status to the message so something like: "Terminal: The canary terminated without an error message".

Reply:
fixed